<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta http-equiv="x-ua-compatible" content="IE=edge"/>
<title>Test results - ServiceTest</title>
<link href="../css/base-style.css" rel="stylesheet" type="text/css"/>
<link href="../css/style.css" rel="stylesheet" type="text/css"/>
<script src="../js/report.js" type="text/javascript"></script>
</head>
<body>
<div id="content">
<h1>ServiceTest</h1>
<div class="breadcrumbs">
<a href="../index.html">all</a> &gt; 
<a href="../packages/ru.mai.lessons.rpks.html">ru.mai.lessons.rpks</a> &gt; ServiceTest</div>
<div id="summary">
<table>
<tr>
<td>
<div class="summaryGroup">
<table>
<tr>
<td>
<div class="infoBox" id="tests">
<div class="counter">18</div>
<p>tests</p>
</div>
</td>
<td>
<div class="infoBox" id="failures">
<div class="counter">0</div>
<p>failures</p>
</div>
</td>
<td>
<div class="infoBox" id="ignored">
<div class="counter">0</div>
<p>ignored</p>
</div>
</td>
<td>
<div class="infoBox" id="duration">
<div class="counter">5m9.52s</div>
<p>duration</p>
</div>
</td>
</tr>
</table>
</div>
</td>
<td>
<div class="infoBox success" id="successRate">
<div class="percent">100%</div>
<p>successful</p>
</div>
</td>
</tr>
</table>
</div>
<div id="tabs">
<ul class="tabLinks">
<li>
<a href="#tab0">Tests</a>
</li>
<li>
<a href="#tab1">Standard output</a>
</li>
</ul>
<div id="tab0" class="tab">
<h2>Tests</h2>
<table>
<thead>
<tr>
<th>Test</th>
<th>Duration</th>
<th>Result</th>
</tr>
</thead>
<tr>
<td class="success">testKafkaWriteReadMessage()</td>
<td class="success">17.890s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testPostgreSQLReadValues()</td>
<td class="success">11.532s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringAddNewRule()</td>
<td class="success">33.879s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringContains()</td>
<td class="success">13.190s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringContainsNotContainsTwoRules()</td>
<td class="success">13.614s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringContainsNotEqualsTwoRules()</td>
<td class="success">13.848s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringContainsTwoRules()</td>
<td class="success">13.577s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringEquals()</td>
<td class="success">15.778s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringEqualsContainsTwoRules()</td>
<td class="success">26.093s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringEqualsNotContainsTwoRules()</td>
<td class="success">14.095s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringEqualsNotEqualsTwoRules()</td>
<td class="success">12.660s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringEqualsTwoRules()</td>
<td class="success">12.687s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringNotContains()</td>
<td class="success">14.205s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringNotEquals()</td>
<td class="success">15.568s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringNotRules()</td>
<td class="success">24.480s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testServiceFilteringUpdateRule()</td>
<td class="success">34.082s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testStartKafka()</td>
<td class="success">10.634s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testStartPostgreSQL()</td>
<td class="success">11.712s</td>
<td class="success">passed</td>
</tr>
</table>
</div>
<div id="tab1" class="tab">
<h2>Standard output</h2>
<span class="code">
<pre>16:42:45,187 |-INFO in ch.qos.logback.classic.LoggerContext[default] - This is logback-classic version 1.4.5
16:42:45,208 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
16:42:45,211 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [file:/D:/%d0%9f%d1%80%d0%be%d0%b3%d1%80%d0%b0%d0%bc%d0%bc%d1%8b/IntelliJ%20IDEA%20Community%20Edition%202020.3.2/JavaServiceFiltering/build/resources/test/logback.xml]
16:42:45,211 |-WARN in ch.qos.logback.classic.util.DefaultJoranConfigurator@4b45dcb8 - Resource [logback.xml] occurs multiple times on the classpath.
16:42:45,211 |-WARN in ch.qos.logback.classic.util.DefaultJoranConfigurator@4b45dcb8 - Resource [logback.xml] occurs at [file:/D:/%d0%9f%d1%80%d0%be%d0%b3%d1%80%d0%b0%d0%bc%d0%bc%d1%8b/IntelliJ%20IDEA%20Community%20Edition%202020.3.2/JavaServiceFiltering/build/resources/test/logback.xml]
16:42:45,211 |-WARN in ch.qos.logback.classic.util.DefaultJoranConfigurator@4b45dcb8 - Resource [logback.xml] occurs at [file:/D:/%d0%9f%d1%80%d0%be%d0%b3%d1%80%d0%b0%d0%bc%d0%bc%d1%8b/IntelliJ%20IDEA%20Community%20Edition%202020.3.2/JavaServiceFiltering/build/resources/main/logback.xml]
16:42:45,326 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - Processing appender named [CONSOLE]
16:42:45,326 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - About to instantiate appender of type [ch.qos.logback.core.ConsoleAppender]
16:42:45,374 |-WARN in ch.qos.logback.core.ConsoleAppender[CONSOLE] - This appender no longer admits a layout as a sub-component, set an encoder instead.
16:42:45,374 |-WARN in ch.qos.logback.core.ConsoleAppender[CONSOLE] - To ensure compatibility, wrapping your layout in LayoutWrappingEncoder.
16:42:45,374 |-WARN in ch.qos.logback.core.ConsoleAppender[CONSOLE] - See also http://logback.qos.ch/codes.html#layoutInsteadOfEncoder for details
16:42:45,375 |-INFO in ch.qos.logback.classic.model.processor.LoggerModelHandler - Setting level of logger [ru.mai.lessons.rpks] to INFO
16:42:45,378 |-INFO in ch.qos.logback.classic.model.processor.LoggerModelHandler - Setting additivity of logger [ru.mai.lessons.rpks] to false
16:42:45,378 |-INFO in ch.qos.logback.core.model.processor.AppenderRefModelHandler - Attaching appender named [CONSOLE] to Logger[ru.mai.lessons.rpks]
16:42:45,379 |-INFO in ch.qos.logback.classic.model.processor.RootLoggerModelHandler - Setting level of ROOT logger to ERROR
16:42:45,379 |-INFO in ch.qos.logback.core.model.processor.AppenderRefModelHandler - Attaching appender named [CONSOLE] to Logger[ROOT]
16:42:45,379 |-INFO in ch.qos.logback.core.model.processor.DefaultProcessor@7216fb24 - End of configuration.
16:42:45,380 |-INFO in ch.qos.logback.classic.joran.JoranConfigurator@2072acb2 - Registering current configuration as safe fallback point

16:42:58.718 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Bootstrap.servers: PLAINTEXT://localhost:53058
16:42:58.721 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:42:58.944 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Creating topics
16:42:59.737 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Created topics
16:42:59.738 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Sending message
16:42:59.876 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Sent message
16:42:59.876 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Consumer subscribe
16:42:59.877 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Consumer start reading
16:42:59.992 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:43:01.113 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 9
16:43:02.187 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:43:15.038 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:43:15.949 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:43:16.409 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:43:16.443 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:43:16.765 [pool-2-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:43:17.111 [pool-2-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:43:17.144 [pool-2-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:43:17.146 [pool-2-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:43:17.148 [pool-2-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:43:17.149 [pool-2-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:43:17.151 [pool-2-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:43:17.823 [pool-2-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:43:17.826 [pool-2-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-3, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:43:17.836 [pool-2-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:43:30.467 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:43:30.485 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:43:31.289 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:43:31.365 [pool-4-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:43:31.370 [pool-4-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:43:32.404 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 9
16:43:33.526 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 8
16:43:34.646 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 7
16:43:35.763 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 6
16:43:36.882 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 5
16:43:37.996 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 4
16:43:39.117 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 3
16:43:40.242 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 2
16:43:41.355 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 1
16:43:42.370 [pool-4-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-5, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:43:42.373 [pool-4-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:43:54.277 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:43:54.298 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:43:54.709 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:43:54.725 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:43:54.732 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:43:54.763 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:43:54.997 [pool-6-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:43:55.040 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:43:55.041 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:43:55.060 [pool-6-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:43:55.061 [pool-6-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:43:55.061 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:43:55.063 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:43:55.063 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:43:55.063 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:43:55.063 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:43:55.064 [pool-6-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:43:55.065 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:43:56.042 [pool-6-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:43:56.043 [pool-6-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-7, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:43:56.045 [pool-6-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:43:56.626 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:08.273 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:44:08.290 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:44:08.731 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:44:08.745 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:44:08.970 [pool-8-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:44:09.036 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:44:09.037 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:09.037 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:44:09.038 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:44:09.038 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:09.039 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:09.040 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:09.040 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:44:10.004 [pool-8-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:44:10.004 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:44:10.007 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:44:10.010 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Wait until application updated rules from DB
16:44:21.074 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:26.629 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:30.017 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:44:30.020 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:30.022 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:44:30.026 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:44:30.028 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:30.033 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:30.039 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:30.042 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:44:30.042 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:44:30.058 [pool-8-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:44:30.058 [pool-8-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-9, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:44:30.060 [pool-8-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:44:34.806 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:41.669 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:44:41.686 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:44:42.140 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:44:42.156 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:44:42.158 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:44:42.183 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:44:42.416 [pool-10-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:44:42.582 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:44:42.583 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains true
16:44:42.609 [pool-10-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:44:42.611 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:42.612 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:42.612 [pool-10-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:44:42.613 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:44:42.613 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:44:42.614 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:43.466 [pool-10-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:44:43.467 [pool-10-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-11, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:44:43.468 [pool-10-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:44:51.087 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:56.645 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:45:04.819 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:45:07.407 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:45:07.426 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:45:07.952 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:45:07.969 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:45:08.236 [pool-13-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:45:08.321 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:45:08.346 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:45:08.347 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:45:08.348 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:45:08.349 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:45:08.350 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:45:08.350 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:45:08.351 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:45:08.778 [pool-9-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:45:09.260 [pool-13-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:45:09.266 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:45:09.270 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:45:09.272 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Wait until application updated rules from DB
16:45:21.097 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:45:22.221 [pool-11-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:45:26.648 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:45:29.278 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:45:29.281 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:45:29.284 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:45:29.286 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:45:29.290 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:45:29.292 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:45:29.298 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:45:29.302 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:45:29.316 [pool-13-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:45:29.317 [pool-13-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-13, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:45:29.319 [pool-13-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:45:34.832 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:45:38.780 [pool-9-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:45:41.613 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:45:41.633 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:45:42.114 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:45:42.131 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:45:42.346 [pool-15-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:45:42.436 [pool-15-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:45:42.437 [pool-15-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:45:42.438 [pool-15-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:45:42.439 [pool-15-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:45:42.439 [pool-15-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:45:42.440 [pool-15-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains true
16:45:43.391 [pool-15-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:45:43.391 [pool-15-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-15, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:45:43.393 [pool-15-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:45:51.111 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:45:52.230 [pool-11-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:45:55.827 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:45:55.847 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:45:56.335 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:45:56.347 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:45:56.348 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:45:56.379 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:45:56.598 [pool-17-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:45:56.655 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:45:56.655 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains true
16:45:56.659 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:45:56.675 [pool-17-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:45:56.675 [pool-17-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:45:56.676 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:45:56.676 [pool-17-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:45:56.676 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:45:56.677 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:45:56.677 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:45:56.677 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:45:57.632 [pool-17-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:45:57.633 [pool-17-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-17, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:45:57.634 [pool-17-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:46:04.833 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:08.029 [pool-14-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:08.796 [pool-9-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:09.104 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:46:09.121 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:46:09.614 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:46:09.628 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:46:09.830 [pool-19-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:46:09.904 [pool-19-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:46:09.922 [pool-19-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:46:09.923 [pool-19-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:46:09.923 [pool-19-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:46:09.924 [pool-19-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:46:10.872 [pool-19-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:46:10.873 [pool-19-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-19, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:46:10.875 [pool-19-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:46:21.118 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:22.177 [pool-16-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:22.238 [pool-11-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:26.666 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:34.841 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:34.939 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:46:34.959 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:46:35.508 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:46:35.526 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:46:35.529 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:46:35.555 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:46:35.818 [pool-21-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:46:35.920 [pool-21-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:46:35.921 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:46:35.921 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:46:35.921 [pool-21-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:46:35.922 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:46:35.922 [pool-21-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:46:35.922 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:46:35.923 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:46:35.924 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:46:35.924 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:46:35.924 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:46:35.924 [pool-21-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:46:35.925 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:46:35.925 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:46:35.925 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:46:35.925 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:46:36.411 [pool-18-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:36.837 [pool-21-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:46:36.838 [pool-21-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-21, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:46:36.840 [pool-21-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:46:38.033 [pool-14-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:38.808 [pool-9-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:49.673 [pool-20-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:51.101 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:46:51.119 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:51.119 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:46:51.583 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:46:51.596 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:46:51.799 [pool-23-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:46:51.871 [pool-23-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:46:51.871 [pool-23-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:46:51.872 [pool-23-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:46:51.872 [pool-23-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:46:51.872 [pool-23-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:46:52.184 [pool-16-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:52.247 [pool-11-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:46:52.837 [pool-23-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:46:52.838 [pool-23-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-23, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:46:52.839 [pool-23-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:46:56.676 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:04.271 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:47:04.283 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:47:04.842 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:06.425 [pool-18-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:08.046 [pool-14-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:08.823 [pool-9-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:15.269 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:47:15.285 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:47:15.596 [pool-22-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:15.751 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:47:15.770 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:47:15.772 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:47:15.794 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:47:15.998 [pool-26-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:47:16.071 [pool-26-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:47:16.071 [pool-26-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:47:16.094 [pool-26-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:47:16.094 [pool-26-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:47:16.095 [pool-26-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:47:16.095 [pool-26-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:47:16.095 [pool-26-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:47:16.095 [pool-26-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:47:16.097 [pool-26-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:47:17.025 [pool-26-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:47:17.025 [pool-26-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-25, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:47:17.026 [pool-26-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:47:19.686 [pool-20-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:21.133 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:22.185 [pool-16-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:22.248 [pool-11-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:26.681 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:27.913 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:47:27.928 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:47:28.423 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:47:28.437 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:47:28.439 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:47:28.468 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:47:28.671 [pool-28-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:47:28.725 [pool-28-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:47:28.725 [pool-28-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:47:28.746 [pool-28-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:47:28.747 [pool-28-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:47:28.747 [pool-28-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:47:28.747 [pool-28-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:47:28.747 [pool-28-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:47:28.748 [pool-28-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:47:28.748 [pool-28-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:47:29.694 [pool-28-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:47:29.695 [pool-28-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-27, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:47:29.696 [pool-28-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:47:31.633 [pool-24-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:34.852 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:36.437 [pool-18-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:38.056 [pool-14-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:38.835 [pool-9-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:41.037 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:47:41.057 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:47:41.445 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:47:41.458 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:47:41.459 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:47:41.488 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:47:41.690 [pool-30-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:47:41.743 [pool-30-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:47:41.744 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:47:41.744 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:47:41.745 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:47:41.745 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:47:41.746 [pool-30-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:47:41.746 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:47:41.746 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:47:42.728 [pool-30-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:47:42.728 [pool-30-thread-1] ERROR o.a.k.clients.consumer.KafkaConsumer - [Consumer clientId=consumer-test_group_consumer-29, groupId=test_group_consumer] Failed to close coordinator
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 16 common frames omitted
16:47:42.730 [pool-30-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:41)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:42)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
		at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.pollNoWakeup(ConsumerNetworkClient.java:321)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.maybeLeaveGroup(AbstractCoordinator.java:1131)
		at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.close(AbstractCoordinator.java:1091)
		at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.close(ConsumerCoordinator.java:1053)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2412)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2379)
		at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2352)
		at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:38)
		... 6 common frames omitted
	Caused by: java.lang.InterruptedException: null
		... 16 common frames omitted
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:47:45.602 [pool-22-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:49.702 [pool-20-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:51.145 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:52.189 [pool-16-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:47:52.250 [pool-11-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
</pre>
</span>
</div>
</div>
<div id="footer">
<p>
<div>
<label class="hidden" id="label-for-line-wrapping-toggle" for="line-wrapping-toggle">Wrap lines
<input id="line-wrapping-toggle" type="checkbox" autocomplete="off"/>
</label>
</div>Generated by 
<a href="http://www.gradle.org">Gradle 7.2</a> at 20 июн. 2023 г., 16:47:56</p>
</div>
</div>
</body>
</html>
